plugins/modules/iosxr_bgp_global.py

```diff
@@ -0,0 +1,929 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2021 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+"""
+The module file for iosxr_bgp_global
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: iosxr_bgp_global
+short_description: Manages BGP global resource module.
+description:
+- This module configures and manages the attributes of BGP global on Cisco IOS-XR platforms.
+version_added: 1.3.0
```

Comment:
This should be 2.0.0 since we're doing a major release next.

Reply:
ack